bin/ofs/src/fuse.rs

```diff
@@ -354,6 +354,10 @@ impl PathFilesystem for Fuse {
             name
         );
 
+        if !self.op.info().full_capability().rename {
+            return Err(Errno::from(libc::EACCES))?;
```

Comment:
We should return `ENOTSUP` instead?

Reply:
`Permission denied` or `Operation not supported`.  `Operation not supported`  is better.